admin/partials/digital-climate-strike-wordpress-admin-display.php

```diff
@@ -10,17 +10,17 @@
         <?php
         //Grab all options
         $options = get_option($this->plugin_name);
-        $show_digital_strike_widget = $this->field_is_set($options, 'show_digital_strike_widget') ? 1 : 0;
+        $show_digital_strike_widget = $this->field_is_set($options, 'show_digital_strike_widget') ? self::ENABLE : self::DISABLE;
         $language = $this->field_is_set($options, 'language') ? esc_attr($options['language']) : 'en';
-        $force_full_page_widget = ( isset( $options['force_full_page_widget'] ) && ! empty( $options['force_full_page_widget'] ) ) ? 1 : 0;
-        $always_show_widget = ( isset( $options['always_show_widget'] ) && ! empty( $options['always_show_widget'] ) ) ? 1 : 0;
-        $iframe_host = ( isset( $options['iframe_host'] ) && ! empty( $options['iframe_host'] ) ) ? esc_attr($options['iframe_host']) : 'https://assets.digitalclimatestrike.net';
-        $cookie_expiration_days = ( isset( $options['cookie_expiration_days'] ) && ! empty( $options['cookie_expiration_days'] ) ) ? esc_attr($options['cookie_expiration_days']) : 1;
-        $disable_google_analytics = ( isset( $options['disable_google_analytics'] ) && ! empty( $options['disable_google_analytics'] ) ) ? 1 : 0;
-        $show_close_button_on_full_page_widget = ( isset( $options['show_close_button_on_full_page_widget'] ) && ! empty( $options['show_close_button_on_full_page_widget'] ) ) ? 1 : 0;
+        $force_full_page_widget = $this->field_is_set($options, 'force_full_page_widget') ? self::ENABLE : self::DISABLE;
+        $always_show_widget = $this->field_is_set($options, 'always_show_widget') ? self::ENABLE : self::DISABLE;
+        $iframe_host = $this->field_is_set($options, 'iframe_host') ? esc_attr($options['iframe_host']) : 'https://assets.digitalclimatestrike.net';
```

Comment:
should we extract the iFrame host string to . constant as well?

Reply:
Agree. Fixed as well.